examples/text_labels_and_annotations/text_alignment.py

```diff
@@ -47,24 +49,24 @@
         verticalalignment='top',
         transform=ax.transAxes)
 
-ax.text(left, 0.5*(bottom + top), 'right center',
+ax.text(left, 0.5 * (bottom + top), 'right center',
```

Comment:
Actually it does, see https://www.python.org/dev/peps/pep-0008/#other-recommendations.